index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+
+<body>
+    <h1>Genotify</h1>
+    <form>
+        Search for:
+        <br>
+        <input type="text" id="query" placeholder="CDK2, cyclin kinase 2, etc">
+    </form>
+    <br>
+    <button id="searchButton" type="button">Search</button>
+    <p id="basics"></p>
+    <br>
+    <p id="summary"></p>
+    <script>
+    (function() {
+        var httpRequest;
+        document.getElementById("searchButton").addEventListener('click', makeQuery);
+
+        function makeQuery() {
+            httpRequest = new XMLHttpRequest();
```

Comment:
Consider replacing XMLHttpRequest with [`fetch`](https://developers.google.com/web/updates/2015/03/introduction-to-fetch). On top of being newer, Fetch's syntax is superior and it is generally less of a hassle to work with.

[Here's where I use fetch in one of my projects.](https://github.com/melalawi/GlobalTwitchEmotes/blob/967c2b0866bfb886c6161010d02de568f16313ac/src/common/background/js/httpRequest.js#L11-L24)